### PR TITLE
Used UserAssignedSerializer instead of UserBaseMinimumSerializer for assigned_to_object in patient details

### LINKED ISSUE #2623: Use UserAssignedSerializer for assigned_to_object
### Description:
The `UserBaseMinimum` serializer currently does not include the `alt_phone_number` field, which is necessary for user data which is necessary in patient details page for volunteer . This missing field affects the ability to store and retrieve emergency contact details when using this serializer.

### PR DESCRIPTION
**Pull request by Rishith25**
## Proposed Changes

- Fixes #2623 
- Used UserAssignedSerializer instead of UserBaseMinimumSerializer for assigned_to_object in patient details for volunteer contact

### Architecture changes

- Remove this section if not used

## Merge Checklist

- [ ] Tests added/fixed
- [ ] Update docs in `/docs`
- [ ] Linting Complete
- [ ] Any other necessary step

_*Only PR's with test cases included and passing lint and test pipelines will be reviewed*_

@ohcnetwork/care-backend-maintainers @ohcnetwork/care-backend-admins

## Summary by CodeRabbit

- **Improvements**
	- Updated user assignment serialization for patient records
	- Enhanced representation of assigned user details in patient-related data structures

### CODE REVIEW

**Comment by Jacobjeevan** [comment]
@vigneshhari @sainak Can either of you take a look? Thanks

File: care/users/api/serializers/user.py [hunk 1]
```diff
@@ -388,6 +388,7 @@ class Meta:
             "user_type",
             "last_login",
             "read_profile_picture_url",
+            "alt_phone_number"
```
Comment (sainak) [inline care/users/api/serializers/user.py]:
```suggestion
            "phone_number"
```
the issue https://github.com/ohcnetwork/care_fe/issues/9244 requires the emergency contact's phone number, not contact's emergency/alt phone number

**Review by sainak — CHANGES_REQUESTED** [review]

**Review by rithviknishad — APPROVED** [review]
LGTM as it solves the issue, but not sure if this is a security issue since `UserBaseMinumumSerializer` is used pretty much for all places where just the user's minimum information is required without revealing full details of the user itself.

cc: @sainak @vigneshhari

**Comment by Rishith25** [comment]
@vigneshhari I have used UserAssignedSerializer instead of UserBaseMinimumSerializer.
Can you once look into it.

**Pull request closed without merging**